big_tests/tests/mod_event_pusher_rabbit_SUITE.erl

```diff
@@ -419,6 +419,9 @@ group_chat_message_received_event_properly_formatted(Config) ->
               %% GIVEN users in room
               escalus:send(Alice, muc_helper:stanza_muc_enter_room(Room, nick(Alice))),
               escalus:send(Bob, muc_helper:stanza_muc_enter_room(Room, nick(Bob))),
+              % wait for all room stanzas to be processed
+              escalus:wait_for_stanzas(Alice, 3),
```

Comment:
Why are we waiting for 3 stanzas?

Reply:
That's how many stanzas the room sends as an answer:
```
alicE94.191079@localhost/res1 out
<presence to='muc_publish@muc.localhost/alicE94.191079'>
	<x xmlns='http://jabber.org/protocol/muc'/>
</presence>

alicE94.191079@localhost/res1 in
<presence from='muc_publish@muc.localhost/alicE94.191079' to='alicE94.191079@localhost/res1' xml:lang='en'>
	<x xmlns='http://jabber.org/protocol/muc#user'>
		<item jid='alicE94.191079@localhost/res1' affiliation='owner' role='moderator'/>
		<status code='110'/>
		<status code='100'/>
		<status code='201'/>
	</x>
</presence>

alicE94.191079@localhost/res1 in
<message from='muc_publish@muc.localhost' to='alicE94.191079@localhost/res1' type='groupchat'>
	<subject/>
	<body/>
</message>

alicE94.191079@localhost/res1 in
<presence from='muc_publish@muc.localhost/bOb94.191079' to='alicE94.191079@localhost/res1' xml:lang='en'>
	<x xmlns='http://jabber.org/protocol/muc#user'>
		<item jid='bOb94.191079@localhost/res1' affiliation='none' role='participant'/>
	</x>
</presence>
```

Reply:
For example: https://github.com/esl/MongooseIM/blob/6715faa6c82e4965d12e0db3940a0d28b19a4c27/big_tests/tests/muc_SUITE.erl#L685-L690

Reply:
Thanks :)